src/lib/route-broadcaster.js

```diff
@@ -60,21 +72,33 @@ class RouteBroadcaster {
   removeExpiredRoutesSoon () {
     setTimeout(() => {
       try {
-        this.routingTables.removeExpiredRoutes()
+        let lostLedgerLinks = this.routingTables.removeExpiredRoutes()
+        this.markLedgersUnreachable(lostLedgerLinks)
       } catch (err) {
         log.warn('removing expired routes failed')
         log.debug(err)
       }
-      this.removeExpiredRoutesSoon()
     }, this.routeCleanupInterval)
   }
 
+  markLedgersUnreachable (lostLedgerLinks) {
+    if (lostLedgerLinks.length > 0) log.info('detected lostLedgerLinks:', lostLedgerLinks)
+    lostLedgerLinks.map((unreachableLedger) => { this.detectedDown.add(unreachableLedger) })
+  }
+  _currentEpoch () {
+    return this.routingTables.publicTables.currentEpoch
+  }
+  _endEpoch () {
+    this.routingTables.publicTables.incrementEpoch()
+  }
+
   broadcastSoon () {
     defer.setTimeout(function * () {
       try {
         this.routingTables.removeExpiredRoutes()
-        yield this.reloadLocalRoutes()
+        yield this.reloadLocalRoutes() // todo? remove
```

Comment:
I guess it wouldn't, judging by your response (that's why there was a question mark).  I thought at the time that they might be reloaded because they expired, along with everything else.